### DISCUSSION TITLE
Implement support for finding the highest period as well?

### DISCUSSION THREAD

**Discussion by twosh** [dala318/nordpool_planner#15] [Ideas]
Hi!
Started using your plug-in today! Great job, making it easy to find the lowest average energy cost. 👏

Would it by any chance be possible for you to also add the functionality for finding the most expensive time period?

For me, I just setup the plug-in and will use it to charge my 10kWh battery during the cheapest period of time. Tonight this would happen at 1:00-4:00 am at about 3300 W/h. Awesome i thought! 😊 

However, then I noticed that the next consecutive hours are quite cheap as well, especially compared to the most expensive hours later in the morning, and I would really want to save the battery energy to the three most expensive hours during the day and then discharge. Now, my battery will be empty already during pretty cheap hours around 4:00-7:00 since with this cold my heating draws around 3kW/h. Between 8:00-10:00 it will be the most expensive period, and it would of course be the most gain for me (and Europe balancing the net) if I could use my energy during that time period instead.

What do you day? Does it make sense? 😅

Cheers,
Tim

**Comment by dala318** [comment]
Good idea. Created #30 as an enhancement. Will likely not be done before I've found time to do #23 and #27 as those will need some major re-work and could enable this feature.

**Comment by dala318** [comment]
Is a part of beta-release v2.0.0b3
